CRM/Utils/System/Standalone.php

```diff
@@ -211,16 +263,20 @@ public static function currentPath() {
    * @param string $password
    *   The password for the above user.
    * @param bool $loadCMSBootstrap
-   *   Load cms bootstrap?.
+   *   Not used in Standalone context
    * @param string $realPath
-   *   Filename of script
+   *   Not used in Standalone context
    *
    * @return array|bool
    *   [contactID, ufID, unique string] else false if no auth
    * @throws \CRM_Core_Exception.
    */
   public function authenticate($name, $password, $loadCMSBootstrap = FALSE, $realPath = NULL) {
-    return Security::singleton()->authenticate($name, $password, $loadCMSBootstrap, $realPath);
+    $authxLogin = authx_login(['flow' => 'login', 'cred' => 'Basic ' . base64_encode("{$name}:{$password}")]);
```

Comment:
@totten picked up that this will mean authx policies are applied to logins here.

This would mean it was denied if you removed `password` from authx allowed credentials. But this is required on Standalone anyway (as is required for web log in).